config/jobs/kubernetes/sig-testing/godeps.yaml

```diff
@@ -14,16 +14,16 @@ presubmits:
       - name: main
         command:
         - runner.sh
-        - make
         image: gcr.io/k8s-testimages/kubekins-e2e:v20181205-915278e90-master
         args:
+        - make
         - verify
-        - WHAT="godeps staging-godeps godep-licenses"
+        - WHAT=\'godeps staging-godeps godep-licenses\'
```

Comment:
I do, because I need this variable to be a space separated string when it gets interpreted by bash. That said, just sticking this in the environment, rather than mucking with escaping in runner.sh seems like the way to go.